modules/openid_auth/app/controllers/openid_auth/v0/okta_controller.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require_dependency 'openid_auth/application_controller'
+
+module OpenidAuth
+  module V0
+    class OktaController < ApplicationController
+      skip_before_action :authenticate
+
+      def metadata
```

Comment:
This looks like dead code, don't need a SAML metadata endpoint in the Okta hook.